MinionMovies/MovieListController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ViewController.swift
+//  MinionMovies
+//
+//  Created by Vitor Costa on 26/02/20.
+//  Copyright © 2020 Vitor Costa. All rights reserved.
+//
+
+import UIKit
+
+class MovieListController: UIViewController {
+
+    let moviesImages: [String] = ["jumanji", "reiLeão"]
```

Comment:
Quando for pegar os dados da API tu vai acabar tendo que refazer bastante coisa, vou criar o endpoint pra vocês poderem pegar os dados de lá

Reply:
Aham. Eu tinha explicado essa parte no pr anterior e esqueci de replicar nesse. Eu só tinha feito isso pra teste mesmo.